src/components/Pill/PillButton/__tests__/PillButton.spec.tsx

```diff
@@ -8,52 +10,78 @@ describe('<PillButton> component', () => {
     const onClearMock = jest.fn();
     const name = 'Pill name';
     const content = 'This pill is in use';
+    const downArrowLabel = 'down arrow';
+    const upArrowLabel = 'up arrow';
+    const clearLabel = 'clear label';
 
-    let wrapper;
+    let view: RenderResult;
+
+    const getButtonByName = (inputName) => {
+        return screen.getByRole('button', { name: `${inputName}` });
+    };
 
     afterEach(() => {
```

Comment:
I don't think we need it with the current version of Jest. Can you check if this can be removed or not?

Reply:
yes i've removed it